bundesliga_app/views.py

```diff
@@ -1,3 +1,80 @@
-from django.shortcuts import render
+from django.views.generic.base import TemplateView
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.decorators import login_required
+from django.urls import reverse
+from django.utils.decorators import method_decorator
+from django.views.generic.list import ListView
+from eventbrite import Eventbrite
+from bundesliga_app.models import Event
+from django.http import HttpResponseRedirect
+from .utils import get_auth_token
 
-# Create your views here.
+
+@method_decorator(login_required, name='dispatch')
+class HomeView(ListView, LoginRequiredMixin):
+
+    """ This is the index view. Here we display all the banners that the user
+    has created """
+
+    template_name = 'index.html'
+
+    queryset = Event.objects.all()
+
+
+@method_decorator(login_required, name='dispatch')
+class SelectEvents(TemplateView, LoginRequiredMixin):
+
+    """ This is the index view. Here we display all the banners that the user
+    has created """
+
+    template_name = 'organizer/select_events.html'
+
+    def _get_event(self):
+        eventbrite = Eventbrite(
+            get_auth_token(self.request.user),
+        )
+        return [
+            event
+            # Status : live, draft, canceled, started, ended, all
+            for event in eventbrite.get(
+                '/users/me/owned_events/?status=live,draft'
+            )['events']
+        ]
+
+    def post(self, *args, **kwargs):
+        # import ipdb; ipdb.set_trace()
```

Comment:
If is possible, try to remove debugger from commits

Reply:
in another commit is going to be removed! thanks!